morpho-checkpoint-node/tests/Test/Morpho/Examples.hs

```diff
@@ -175,32 +175,95 @@ exampleAnnTip =
 exampleApplyTxErr :: MorphoError TestBlock
 exampleApplyTxErr = MorphoInvalidSignature exampleVote
 
-exampleNodeConfig :: NodeConfiguration
-exampleNodeConfig =
+minimalConfig :: NodeConfiguration
+minimalConfig =
   NodeConfiguration
     { ncProtocol = MockedBFT,
-      ncNodeId = CoreId $ CoreNodeId 0,
+      ncNodeId = CoreNodeId 0,
       ncNumCoreNodes = 1,
-      ncReqNetworkMagic = RequiresMagic,
       ncNetworkMagic = 12345,
-      ncSystemStart = Just $ SystemStart $ posixSecondsToUTCTime $ realToFrac (1234566789 :: Integer),
+      ncSystemStart = Nothing,
       ncSecurityParameter = 3,
       ncStableLedgerDepth = 2,
       ncLoggingSwitch = True,
-      ncTraceOpts = exampleTraceOptions,
-      ncLogMetrics = True,
-      ncViewMode = SimpleView,
-      ncUpdate = Update $ LastKnownBlockVersion 0 2 0,
+      ncTraceOpts =
+        TraceOptions
+          { traceVerbosity = NormalVerbosity,
+            traceChainDB = True,
+            traceChainSyncClient = True,
+            traceChainSyncHeaderServer = True,
+            traceChainSyncBlockServer = True,
+            traceBlockFetchDecisions = True,
+            traceBlockFetchClient = True,
+            traceBlockFetchServer = True,
+            traceTxInbound = True,
+            traceTxOutbound = True,
+            traceLocalTxSubmissionServer = True,
+            traceMempool = True,
+            traceForge = True,
+            traceChainSyncProtocol = True,
+            traceBlockFetchProtocol = True,
+            traceBlockFetchProtocolSerialised = True,
+            traceTxSubmissionProtocol = True,
+            traceLocalChainSyncProtocol = True,
+            traceLocalTxSubmissionProtocol = True,
+            traceLocalStateQueryProtocol = True,
+            traceIpSubscription = True,
+            traceDnsSubscription = True,
+            traceDnsResolver = True,
+            traceErrorPolicy = True,
+            traceMux = True,
+            traceHandshake = True,
+            traceLedgerState = True,
+            tracePoWNodeRpc = True,
+            traceTimeTravelError = True
+          },
       ncTimeslotLength = mkSlotLength 5,
       ncSnapshotsOnDisk = 60,
       ncSnapshotInterval = 60,
-      ncPoWBlockFetchInterval = Just 5000000,
+      ncPoWBlockFetchInterval = 1000000,
       ncPoWNodeRpcUrl = "http://127.0.0.1:8546",
       ncPrometheusPort = 13788,
       ncCheckpointInterval = 4,
       ncRequiredMajority = 1,
+      ncFedPubKeys = [],
+      ncNodePrivKeyFile = "abc",
+      ncTopologyFile = TopologyFile "/path/to/topo.json",
+      ncDatabaseDir = DbFile "/path/to/db",
+      ncSocketFile = SocketFile "/path/to/socket",
+      ncNodeHost = NodeHostAddress Nothing,
+      ncNodePort = 1234,
+      ncValidateDb = False
+    }
+
+exampleNodeConfig :: NodeConfiguration
+exampleNodeConfig =
```

Comment:
All this stuff could be a bit easier to maintain as record updates off the "minimal" one, perhaps. (Similarly for TraceOptions.)

Reply:
I think for tests it's fine to be a bit repetitive. In this case, these values match the ones in `morpho-checkpoint-node/tests/configuration/Golden/Config.yaml`, so if you update a test case, you'll only have to update these two locations, without having to think about how the tests are DRY'd.